### ISSUE TITLE
Could you give a hugging face hps v2 predictor checkpoint address？

### ISSUE THREAD

**Issue by qiuyang163** [tgxs002/HPSv2#4]
HK onedriver is too unstable for us
![image](https://github.com/tgxs002/HPSv2/assets/8520472/ec4bc306-80bb-43bd-8e5d-bc9ebe6152c5)

**Comment by qiuyang163** [comment]
@tgxs002

**Comment by tgxs002** [comment]
Thank you for your interest in our work! I will prepare one and tell you when it's ready.

**Comment by qiuyang163** [comment]
Thank you，We recently want to try HPS V2 to increase the alignment text-to-image model

**Comment by tgxs002** [comment]
BTW, maybe you can try clicking the download button in the top left corner?
![1688961460592](https://github.com/tgxs002/HPSv2/assets/38932123/688d9569-f7db-4128-b948-8585159ff449)

**Comment by qiuyang163** [comment]
Yes I can click the download button，but after I clicked it, nothing happen

**Comment by KeqiangSun** [comment]
The checkpoint is now available on [HuggingFace](https://huggingface.co/xswu/HPSv2). Check it out😄
